src/main/java/io/opencensus/integration/jdbc/Observability.java

```diff
@@ -218,29 +193,34 @@ void end() {
       if (closed) return;
 
       try {
-        // Record the number of calls of the made method.
-        recordTaggedStat(KEY_METHOD, this.method, MEASURE_CALLS, 1L);
+        // Finally record the latency of the entire call,
+        // as well as "status": "OK" for non-error calls.
+        Map<TagKey, TagValue> tags = new HashMap<>();
+        tags.put(KEY_METHOD, TagValue.create(this.method));
+
+        if (recordedError == null) {
+          tags.put(KEY_STATUS, VALUE_OK);
+        } else {
+          tags.put(KEY_ERROR, TagValue.create(recordedError));
+          tags.put(KEY_STATUS, VALUE_ERROR);
+        }
 
         long totalTimeNs = System.nanoTime() - this.startTimeNs;
         double timeSpentMs = ((double) totalTimeNs) / 1e6;
 
-        // Finally record the latency of the entire call.
-        recordTaggedStat(KEY_METHOD, this.method, MEASURE_LATENCY_MS, timeSpentMs);
+        // Now finally record all the stats the same tags.
+        recordStatWithTags(MEASURE_LATENCY_MS, timeSpentMs, tags);
+        recordStatWithTags(MEASURE_CALLS, 1L, tags);
```

Comment:
Don't need to record this. if you need the count you can install a view with a count aggregation on latency.

Reply:
Thank you @bogdandrutu PTAL!